packages/react/src/components/Button/Button.tsx

```diff
@@ -77,6 +77,9 @@ export interface ButtonProps
   /** A button can emphasize that it represents an alternative action. */
   secondary?: boolean
 
+  /** A button that represents an alternative action and also has contrast against dark backgrounds */
```

Comment:
We don't want to suggest a styling concepts in the properties description. You can see how this is describe in Semantic for the basic button: `The basic button has a subtle appearance.` - it is not describing the styling, but the purpose of the new term (prop) on the Button